substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/Target_sun_security_ssl_TrustStoreManager.java

```diff
@@ -75,46 +85,161 @@ public void afterRegistration(AfterRegistrationAccess access) {
          * lib/security/blacklisted.certs, so this class must be initialized at image build time.
          * This is the default anyway for code JDK classes, but since this this class is relevant
          * for security we spell it out explicitly.
+         *
+         * Note when a runtime certificate file is specified, we still honor/use the build time
+         * lib/security/blacklisted.certs file
          */
         RuntimeClassInitialization.initializeAtBuildTime(sun.security.util.UntrustedCertificates.class);
-        RuntimeClassInitialization.initializeAtBuildTime(org.jcp.xml.dsig.internal.dom.XMLDSigRI.class);
```

Comment:
This class queries "java.specification.version", but it's not strictly necessary to have it explicitly initialized at buildtime. Your call on whether to keep it here or not